topics/firefox/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: seamonkey, waterfox, palemoon, icecat, iceweasel, torbrowser
+created_by: Mozilla
+display_name: Mozilla Firefox
+github_url: https://github.com/mozilla
+released: September 23, 2002
+short_description: An advanced, free, open-source web browser.
```

Comment:
Could you rephrase this as a full sentence and remove the hyphen from "open-source"? Maybe "Firefox is an advanced, free, open source web browser."